Gopkg.toml

```diff
@@ -10,6 +10,10 @@ required = [
   "sigs.k8s.io/controller-tools/pkg/crd/generator",
```

Comment:
Please update the commit message to include a message and description and follow the 50/72 rule.